src/components/Description/DescriptionLLM/index.tsx

```diff
@@ -1,99 +1,101 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 
 import cssBinder from 'styles/cssBinder';
 import globalStyles from 'styles/interpro-vf.css';
 import localStyles from './style.css';
 import fonts from 'EBI-Icon-fonts/fonts.css';
 import Link from 'components/generic/Link';
 
-import Description from '..';
 import config from 'config';
 
 const css = cssBinder(globalStyles, localStyles, fonts);
 
 type Props = {
   accession: string;
-  text: string;
 };
 
-class DescriptionLLM extends PureComponent<Props> {
-  render() {
-    const accession = this.props.accession || '';
-    const text = this.props.text || '';
-    if (accession.length === 0 || text.length === 0) return null;
+const DescriptionLLM = ({ accession }: Props) => {
+  if ((accession || '').length === 0) return null;
 
-    return (
-      <>
-        <div className={css('row')}>
-          <div className={css('columns', 'large-12')}>
-            <div
-              className={css('callout', 'warning')}
-              style={{
-                display: 'flex',
-                alignItems: 'center',
-              }}
+  return (
+    <div className={css('vf-stack', 'vf-stack--400')}>
+      <div
+        className={css('callout', 'warning')}
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+        }}
+      >
+        <span
+          style={{
+            fontSize: '2em',
+            color: '#664d03',
+            paddingRight: '1rem',
+          }}
+          className={css(
+            'small',
+            'icon',
+            'icon-common',
+            'icon-exclamation-triangle',
+          )}
+        />{' '}
+        <p>
+          Some paragraphs of this description have been automatically generated
```

Comment:
_Some paragraphs_ implies more than one paragraph. I tried to come up with a phrasing that works in all cases:
 
```suggestion
          The description below includes sections that have been automatically generated
```